addons/pos_loyalty/static/src/overrides/models/pos_order.js

```diff
@@ -1474,22 +1476,20 @@ patch(PosOrder.prototype, {
                 return _t("That coupon code has already been scanned and activated.");
             }
             const customerId = this.get_partner() ? this.get_partner().id : false;
-            const { successful, payload } = await this.pos.data.call(
-                "pos.config",
-                "use_coupon_code",
-                [
-                    [this.config.id],
-                    code,
-                    this.date_order,
-                    customerId,
-                    this.pricelist_id ? this.pricelist_id.id : false,
-                ]
-            );
+            const call = this.pos.data.call; // eslint-disable-line no-restricted-syntax
+            const { successful, payload } = await call("pos.config", "use_coupon_code", [
+                [this.config.id],
```

Comment:
```suggestion
                [this.config_id?.id],
```
Looks like I also missed this when porting pos_loyalty.